components/brave_rewards/resources/extension/brave_rewards/_locales/en_US/messages.json

```diff
@@ -438,5 +430,29 @@
   "pendingContributionTitle": {
     "message": "Pending Contribution",
     "description": "Notification title for pending contribution type"
+  },
+  "grantFinishTitleUGP": {
+    "message": "It’s your lucky day!",
+    "description": "Cosmetic text upon captcha success"
+  },
+  "grantFinishTextUGP": {
+    "message": "Your token grant is on its way.",
+    "description": "Cosmetic text indicating grant funds are on the way"
+  },
+  "grantFinishTokenTitleUGP": {
+    "message": "Free Token Grant",
+    "description": "Token title on success screen"
+  },
+  "grantFinishTitleAds": {
+    "message": "Brave Ads Rewards!",
+    "description": "Cosmetic text upon captcha success for ads"
+  },
+  "grantFinishTextAds": {
+    "message": "Your rewards grant from Brave Ads is on its way",
```

Comment:
needs a full-stop to be consistent with UGP

Reply:
done